lib/bolt_server/transport_app.rb

```diff
@@ -45,9 +45,30 @@ def initialize(config)
 
       @file_cache = BoltServer::FileCache.new(@config).setup
 
+      # This is needed until the PAL is threadsafe.
+      @pal_mutex = Mutex.new
+      # These are only needed for the `/plans` endpoint.
+      require 'puppet'
+      require 'bolt/pal'
+      require 'bolt_server/pe/pe_pal'
+
       super(nil)
     end
 
+    get '/plans' do
```

Comment:
nit: I think this get route should be defined down with the others in the file and not with the other supporting methods.